tests/step-005.sh

```diff
@@ -15,14 +15,14 @@ cd /tmp/shared/hooks-005.git &&
 if echo "$EXTRA_INSTALL_ARGS" | grep -q "use-core-hookspath"; then
     echo 'n
 y
-/tmp/shared/hooks-005.git
+file:///tmp/shared/hooks-005.git
```

Comment:
Just checking that these should still work without changes, right?
E.g. we don't need users to set up existing shared hooks again?

Reply:
Or this is the new check where we don't allow local paths as shared repos unless  you make it local due to security concerns?
If that's the case, then it's fine.

Reply:
Correction: Unless you make it global, local is prohibited.
Since I changed the folder nameing, **we need to tell the user to setup shared hooks again**: Do we need post install logic for this update? Or just a warning if the user updates from a earlier commit then this date we show it...

Reply:
So if the user added local shared repos that are in the config (but not checked in through the .shared file) those would be nice to either keep working or auto-upgrade or something. If there are absolute paths without a protocol in the .shared file in a repo, I'm OK to have those broken then they'll see the warning you added.

Reply:
Ok, I ll see if we can setup a legacy transform in the install.sh